build_info/libbdplus-dev.control

```diff
@@ -0,0 +1,19 @@
+Package: libbdplus-dev
+Version: @DEB_LIBBDPLUS_V@
+Architecture: @DEB_ARCH@
+Priority: optional
+Section: Development
+Maintainer: @DEB_MAINTAINER@
+Depends: libbdplus0 (= @DEB_LIBBDPLUS_V@)
```

Comment:
Remove version restriction.

Reply:
Don’t remove the version restriction… dev packages should be attached to their specific shlib version.